README.md

```diff
@@ -81,6 +81,10 @@ There are several settings you can configure to customize the behavior of this e
             <td>flake8.ignorePatterns</td>
             <td><code>[]</code></td>
             <td>Configure [glob patterns](https://docs.python.org/3/library/fnmatch.html) as supported by the fnmatch Python library to exclude files or folders from being linted with Flake8.</td>
+        <tr>
+            <td>flake8.enabled</td>
+            <td><code>true</code></td>
+            <td>Specifies whether to enable or disable linting Python files using Flake8. This setting can be applied globally or at the workspace level. If disabled, the linting server itself will continue to be active and monitor read and write events, but it won't perform linting or expose code actions. </td>
```

Comment:
I think Code Actions is supposed to be capitalized based on other VS Code docs [(ref)](https://code.visualstudio.com/docs/editor/refactoring#_code-actions-quick-fixes-and-refactorings)
```suggestion
            <td>Specifies whether to enable or disable linting Python files using Flake8. This setting can be applied globally or at the workspace level. If disabled, the linting server itself will continue to be active and monitor read and write events, but it won't perform linting or expose Code Actions. </td>
```

Reply:
great catch, thank you!!